sprints/dashboard/utils.py

```diff
@@ -497,23 +497,26 @@ def create_next_sprint(conn: CustomJira, sprints: List[Sprint], cell_key: str, b
     """Creates next sprint for the desired cell."""
     sprints = filter_sprints_by_cell(sprints, cell_key)
     last_sprint = sprints[-1]
-    end_date = parse(last_sprint.endDate)
 
-    future_next_sprint_number = get_sprint_number(last_sprint) + 1
-    future_name_date = (end_date + timedelta(days=1)).strftime(settings.JIRA_API_DATE_FORMAT)
-    future_end_date = end_date + timedelta(days=settings.SPRINT_DURATION_DAYS)
+    start_date = parse(get_sprint_end_date(last_sprint)) + timedelta(days=1)
+    start_date_str = start_date.strftime(settings.JIRA_API_DATE_FORMAT)
+    end_date_str = _get_sprint_end_date(start_date_str)
+
+    sprint_number = get_sprint_number(last_sprint) + 1
+    name_date = start_date.strftime(settings.JIRA_API_DATE_FORMAT)
```

Comment:
This contains the same value as `start_date_str`. Do we really need it?

Reply:
Not at all. Good catch.